<launch>
  <arg name="input_topic_name" default="/pointcloud"/>
  <arg name="output_topic_name" default="/pointcloud_filtered"/>
  <arg name="input_frame" default=""/>
  <arg name="output_frame" default=""/>
  <arg name="dual_return_outlier_filter_param_file" default="$(find-pkg-share autoware_pointcloud_preprocessor)/config/dual_return_outlier_filter_node.param.yaml"/>

  <node pkg="autoware_pointcloud_preprocessor" exec="dual_return_outlier_filter_node" name="dual_return_outlier_filter_node">
    <param from="$(var dual_return_outlier_filter_param_file)"/>
    <remap from="input" to="$(var input_topic_name)"/>
    <remap from="output" to="$(var output_topic_name)"/>
    <param name="input_frame" value="$(var input_frame)"/>
    <param name="output_frame" value="$(var output_frame)"/>
  </node>
</launch>
